modules/__tests__/Media-test.js

```diff
@@ -441,25 +392,21 @@ describe("A <Media> in browser environment", () => {
           </Media>
         );
 
-        renderStrict(element, node, () => {
-          expect(node.innerHTML).toMatch("not matched");
-        });
+        expect(container.innerHTML).toMatch("not matched");
       });
     });
 
     describe("for multiple queries", () => {
       it("initially overwrites defaultMatches with matches from matchMedia", async () => {
-        const element = (
+        const { container } = renderStrict(
           <Media queries={{ matches: "(min-width: 1000px)" }} defaultMatches={{ matches: true }}>
             {({ matches }) =>
               matches ? <div>fully matched</div> : <div>not matched</div>
             }
           </Media>
         );
 
-        renderStrict(element, node, () => {
-          expect(node.innerHTML).toMatch("not matched");
-        });
+        expect(container.innerHTML).toMatch("not matched");
```

Comment:
Note that none of the test logic changed substantially.

On a side note, is it possible to test the two-pass render behavior? I don't have a solid enough understanding of how it should work to be able to write a test against it.